console/core/index.ts

```
@@ -1,3 +1,4 @@
+/* eslint-disable no-barrel-files/no-barrel-files */
```

ESLint

[error] Definition for rule 'no-barrel-files/no-barrel-files' was not found.

(no-barrel-files/no-barrel-files)